src/htmlParser.js

```diff
@@ -0,0 +1,391 @@
+// A stateless HTML Parser written in JavaScript
+// Based on http://ejohn.org/blog/pure-javascript-html-parser/
+// Turns an HTML string into a token iterator.
+
+//TODO(#39)
+/*globals console:false*/
+var supports = (function() {
+  var target = {};
+
+  var html;
+  var work = this.document.createElement('div');
```

Comment:
This fixes my issue and loads postscribe correctly in my webpack package! Thanks! +1 for getting this merged and released asap!